### ISSUE TITLE
Stregforbud on too little quantity

### ISSUE THREAD

**Issue by Zaph-x** [f-klubben/stregsystemet#281]
When you buy more than what is available of a given ware, you are given stregforbud. This does not reflect the actual problem at hand, as it is not the user who has insufficient funds, but rather the TREO who has insufficient inventory.
![image](https://user-images.githubusercontent.com/35431127/136572195-2e9bba7a-6503-4d67-8e2f-592f4dc56d2f.png)
This will inevitably become a problem once the inventory system (#274) is merged into production, as this will update the quantity of a given product, based on the inventory stock.

**Comment by JakobTopholt** [comment]
Surely the inventory system will be made some day :copium:
